packages/react/src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -281,7 +278,30 @@ const defaultProps = {
   breakpointSwitcher: null,
   supportedCardTypes: Object.keys(DASHBOARD_EDITOR_CARD_TYPES),
   renderHeader: null,
-  renderIconByName: null,
+  renderIconByName: (iconName, iconProps) => {
```

Comment:
Could this get defined in its own function? Its pretty big to go inside the defaultProps

Reply:
good idea, I moved it out